src/shared/user-menu.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import Avatar from '@material-ui/core/Avatar'
+import ExpandMore from '@material-ui/icons/ExpandMore'
+
+import Button from '@material-ui/core/Button'
+import Menu from '@material-ui/core/Menu'
+import Tooltip from '@material-ui/core/Tooltip'
+
+import MenuItem from '@material-ui/core/MenuItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import NotebookMenuDivider from '../components/menu/notebook-menu-divider'
+
+export default class UserMenu extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      anchorElement: null,
+      isLoggedIn: props.isAuthenticated,
+    }
+
+    this.handleClick = this.handleClick.bind(this)
+    this.goToProfile = this.goToProfile.bind(this)
+    this.handleMenuClose = this.handleMenuClose.bind(this)
+    this.login = this.login.bind(this)
+    this.logout = this.logout.bind(this)
+  }
+
+  handleClick(event) {
+    this.setState({ anchorElement: event.currentTarget })
+  }
+
+  handleMenuClose() {
+    this.setState({ anchorElement: null })
+  }
+
+  goToProfile() {
+    // maybe we should replace this with Link from react-router-dom
```

Comment:
I think we're not going to be using react router, so may as well remove this comment.